tests/kiwi_images_test/validate_build.pm

```diff
@@ -16,63 +24,47 @@ use warnings;
 use testapi;
 use utils;
 use version_utils "is_sle";
-my $sle_version = get_var('VERSION');
-my @files_list  = ('build.log');
+my $logfile = 'build.log';
 
-# check kiwi build status
-sub check_status {
-    my @builds = ("test-image-docker", "test-image-iso", "test-image-oem", "test-image-pxe", "test-image-vmx");
-    script_run("curl -k https://build.suse.de/project/monitor/QA:Maintenance:Images:$sle_version:kiwi-ng-testing > kiwi_out.html");
-    # sle12sp3 has 2 different builds
-    if (is_sle('<=12-SP3')) {
-        script_run("curl -k https://build.suse.de/project/monitor/QA:Maintenance:Images:$sle_version:kiwi-testing > kiwi_out2.html");
-        push @files_list, 'build2.log';
-    }
-    # parse build.log
-    foreach my $kiwi_build (@builds) {
-        script_run("(grep $kiwi_build kiwi_out.html | grep -q succeeded && echo \"SLE-$sle_version $kiwi_build PASSED\" || 
-        echo \"SLE-$sle_version $kiwi_build FAILED\") >> build.log");
-    }
+sub run {
+    my $sle_version  = get_var('VERSION');
+    my $install_type = get_var('KIWI_OLD');
+    my $exit_status;
```

Comment:
for what is it ?

Reply:
Ok, $exit_status is not needed, $sle_version (12sp3/sp4 ...) is needed. Also, $install_type because there are 2 kiwi, kiwi and kiwi-ng builds.

Reply:
Asking only for exit_status as It was not used anywhere.